src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/QRScanner/ExposureSubmissionQRScannerViewController.swift

```diff
@@ -84,134 +78,71 @@ final class ExposureSubmissionQRScannerViewController: UIViewController {
 		instructionLabel.layer.shadowRadius = 3
 		instructionLabel.layer.shadowOffset = .init(width: 0, height: 0)
 
-		navigationController?.overrideUserInterfaceStyle = .dark
+		// setup video capture layer
+		let captureVideoPreviewLayer = AVCaptureVideoPreviewLayer(session: viewModel.captureSession)
+		captureVideoPreviewLayer.frame = view.bounds
+		captureVideoPreviewLayer.videoGravity = .resizeAspectFill
+		view.layer.insertSublayer(captureVideoPreviewLayer, at: 0)
+		previewLayer = captureVideoPreviewLayer
+	}
 
+	private func setupNavigationBar() {
+		navigationController?.overrideUserInterfaceStyle = .dark
```

Comment:
Is it intended to use always dark mode?

Reply:
The QR Scanner should always appear in dark mode. This was previously set in the storyboard